src/Tizen.System.Feedback/Feedback/Feedback.cs

```diff
@@ -596,7 +601,7 @@ public uint GetCountOfThemeInternal(FeedbackType type)
         }
 
         /// <summary>
-        /// Gets the id of theme selected.
+        /// Gets the current ID of the theme selected from available themes described in the conf file.
```

Comment:
```suggestion
        /// Gets the current id of the theme selected from available themes described in the conf file.
```

Keep consistency, id or ID?

Reply:
I changed as "id" to keep consistency thanks.